src/utils/incentive.ts

```diff
@@ -1,12 +1,15 @@
 import { DatabaseManager } from "@subsquid/hydra-common";
 import {
   Chronicle,
+  Contribution,
   HistoricalIncentive,
   Incentive,
   Parachain,
 } from "../generated/model";
 import { Not } from "typeorm";
 import { ensure } from "./ensure";
+import { calculateCurrentContributionReward } from "./calculateRewards"
+import BigNumber from "bignumber.js";
```

Comment:
Why are we using precision multipliers instead of storing raw bn values and then doing calc with bignumber if we need floats?